.github/workflows/test-codecov.yml

```diff
@@ -0,0 +1,21 @@
+name: Using Codecov for Native Python Connections
+on: [push]
+
+jobs:
+  run:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Set up Python 3.7
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.7
+      - name: Install dependencies and run tests
+        working-directory: airbyte-integrations/connectors
+        run: |
+          pip install -r requirements.txt
+          invoke test
+      - name: Upload coverage to Codecov
+        if: ${{ always() }}
+        uses: codecov/codecov-action@v2
```

Comment:
I'm not sure if we gonna need it after the sonarqube is fully configured. @sergei-solonitcyn what do you think?

Reply:
This is necessary so that we can compare test coverage and display it with annotations in the File Changes tab.